bazel_codegen/lib/src/summaries/analysis_context.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:analyzer/src/context/context.dart' show AnalysisContextImpl;
```

Comment:
@bwilkerson would be a better person to answer this question.  I know he has been trying to make it possible for clients to access all the pieces they need from analyzer without needing to import from src, but I don't know specific details of what work remains to be done.